### PR TITLE
Adding contributing guidelines

### PR DESCRIPTION
**Pull request by marinaserranomontes**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    -  If you think the documentation could be clearer, or you have an alternative
+       implementation of something that may have more advantages, we would love to hear it.
+       -  If its a trivial change, go ahead and send a Pull Request with the changes you have in mind
+       -  If not, open a Github Issue to discuss the idea first.
+
+## Requirements
+
+For a contribution to be accepted:
+
+*  Code must follow existing styling conventions
```
Comment (jeffswartz) [inline CONTRIBUTING.md]: There are no coding conventions defined in this repo. There are eslint files or linting tasks. I would remove this.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+2. Clone/fetch your fork to your local development machine.
+3. Create a new branch (e.g. `issue-12`, `feat.add_foo`, etc) and check it out.
+4. Make your changes and commit them.
+5. Push your new branch to your fork. (e.g. `git push myname issue-12`)
+6. Open a Pull Request from your new branch to the original fork's `master` branch.
+
+## Development Guidelines
+
+1. Follow [installation guidelines](README.md).
+2. Modify App.js in root folder.
```
Comment (jeffswartz) [inline CONTRIBUTING.md]: This file does not exist.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Contributing Guidelines
+
+If you're interested in contributing to this project, here are a few ways to do so:
+
+*  Bug fixes
+    -  If you find a bug, please first report it using Github Issues.
+    -  Issues that have already been identified as a bug will be labelled `bug`.
+    -  If you'd like to submit a fix for a bug, send a Pull Request from your own fork and mention the Issue number.
+        +  Include a test that isolates the bug and verifies that it was fixed.
```
Comment (jeffswartz) [inline CONTRIBUTING.md]: There are no existing tests in the repo.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+*  New Features
+    -  If you'd like to add a feature to the library that doesn't already exist, feel free to describe the feature in a new Github Issue.
```
Comment (jeffswartz) [inline CONTRIBUTING.md]: Please change "the library" to "the samples".

File: CONTRIBUTING.md (hunk 1 above)
Comment (marinaserranomontes) [reply CONTRIBUTING.md]: 👍

File: CONTRIBUTING.md (hunk 3 above)
Comment (marinaserranomontes) [reply CONTRIBUTING.md]: 👍 I will remove that line and in the feature we could consider to add tests as well

File: CONTRIBUTING.md (hunk 4 above)
Comment (marinaserranomontes) [reply CONTRIBUTING.md]: 👍

File: CONTRIBUTING.md (hunk 2 above)
Comment (marinaserranomontes) [reply CONTRIBUTING.md]: 👍

**Pull request merged**